v2_ex46.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import csv
+
+filename = 'text_file.txt'
+
+string_of_words = ''
+
+with open(filename, newline='') as csvfile:
+    linereader = csv.reader(csvfile, delimiter=' ')
+    for rows in linereader:
+        for row in rows:
+            string_of_words += row + ' '
+            list_of_words = string_of_words.split()
+
+unique_words = set(list_of_words)
+
+for word in unique_words:
+    frequency = string_of_words.count(word)
+    final_output = f"{word}: {frequency}\n"
+    print(final_output.strip())
```

Comment:
@paulghaddad The output prints out as it should. However, the most frequently used word should appear first (ie, badger: 7
mushroom: 2
...

I'm having trouble sorting this data so the most frequently used word appears first. thoughts?

Also, how can I replace the number of repeats with aestricks:
Ex: 2 ---> **

It seems like I need to use enumerate to index through each number. But I'm not having success with this. Is enumerate the right way to go?